.github/scripts/deny-git-deps.py

```diff
@@ -11,10 +11,8 @@
 
 from cargo_workspace import Workspace, DependencyLocation
 
+# Dependencies and the crate using them that are known to be bad.
 KNOWN_BAD_GIT_DEPS = {
```

Comment:
Why is it called "bad" if this is the allow list? The name somehow sounds like it is the banned deps?

Reply:
Renamed to `ALLOWED_GIT_DEPS` now.